tests/common/features/value_type.rs

```diff
@@ -0,0 +1,26 @@
+use sea_orm::entity::prelude::*;
+use sea_orm_macros::DeriveValueType;
+
+#[derive(Clone, Debug, PartialEq, Eq, DeriveEntityModel)]
+#[sea_orm(table_name = "value_type")]
+pub struct Model {
+    #[sea_orm(primary_key)]
+    pub id: i32,
+    pub number: Integer,
```

Comment:
The strange bit here is that the StringVec is not included in the Model for testing.
Albeit it only works on Postgres.

Reply:
Currently, the conversion of `DeriveValueType` rely on the pre-defined conversion from the field to `Value`.
I think `Vec<String>` currently does not have a defined conversion to any variant of `Value`, which I suppose it should.
(in json_vec, it convert itself to formatted String as `Value`, which I suppose isn't a good generic approach)
I'm wondering if I should implement `From<Vec<T>> for Value` or adjust the `DeriveValueType` implementation only.

Reply:
I see. Sadly then it means that we can't 'eat our own dog food' for this test case.